src/chrome/content/rules/Kitsu.io.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Not resolved:
+		- www.kitsu.io
+
+-->
+<ruleset name="Kitsu (partial)">
```

Comment:
Since `www` wasn't working I thought I should label it as partial. I can change it if it's not needed.